src/content/en/2022/interop.md

```diff
@@ -1,13 +1,13 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Interoperability
-#TODO - Review and update chapter description
 description: Interoperability chapter of the 2022 Web Almanac covering ...
-authors: [bkardell, meyerweb]
-reviewers: [karlcow, foolip, miketaylr]
+authors: [bkardell]
+reviewers: [miketaylr, meyerweb, foolip]
 analysts: [rviscomi, kevinfarrugia]
-editors: []
+editors: [tunetheweb]
 translators: []
+bkardell_bio: Brian Kardell is developer advocate at <a hreflang="en" href="https://igalia.com">Igalia</a>, standards contributor, <a hreflang="en" href="https://bkardell.com">blogger</a>, and is currently the W3C Advisory Committee Representative for the <a hreflang="en" href="https://openjsf.org/">Open JS Foundation</a>. He was a founder of the Extensible Web Community Group and co-author of <a hreflang="en" href="https://extensiblewebmanifesto.org">The Extensible Web Manifesto</a>.
 results: https://docs.google.com/spreadsheets/d/1w3GzzTNeKxafFODmjDs6OC2dseNEDDKwUV8KeSgRI1Y/
 featured_quote: TODO
 featured_stat_1: TODO
```

Comment:
@bkardell (or reviewers!) can you suggest a featured quote, and 3 stats for the home page to drive people to this chapter?